app/konnect/servicehub/service-versions.md

```diff
@@ -57,10 +65,26 @@ From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.co
 
 1. Click **Create** to save.
 
+
+## Managing a service version lifecycle
+
+Changing service version status will have an impact on its visibility in Dev Portal. 
+
+From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service, then selects its service version which status you would like to adjust. Then follow these steps:
+
+1. From the **Service version actions** drop-down menu, select **Edit version status**.
+
+2. In dialog you can select appropriate status. Next to each status there is a brief explanation which pops up on icon hover.
+
+    1. In some cases there might be an additional step to fulfill.
```

Comment:
@tomaszwylezek What other steps would there be here?

Reply:
In the case of changing the version status into "deprecated" - we have one more step in which the user is displayed information about that there will be - check video here - https://github.com/Kong/khcp-ui/pull/2642